src/pool/srv_pool.c

```diff
@@ -4805,6 +4805,7 @@ ds_pool_evict_handler(crt_rpc_t *rpc)
 			D_GOTO(out_free, rc);
 
 		ds_pool_iv_srv_hdl_invalidate(svc->ps_pool);
+		ds_iv_ns_leader_stop(svc->ps_pool->sp_iv_ns);
```

Comment:
should this be done as first operation before you proceed with tx update and hdl invalidation?

Reply:
oh,  we want server handle be invalidated successfully during pool destroy, otherwise if  iv_ns_leader_stop is being executed first,  server handle invalidation will fail.